pkg/status/cel-evaluator.go

```diff
@@ -83,7 +94,9 @@ func (e *celEvaluator) Evaluate(expression v1alpha1.Expression, objMap map[strin
 
 	// evaluate the expression with the unstructured object
 	result, _, err := prog.Eval(map[string]interface{}{
-		"obj": objMap,
+		sourceObjectKey: objMap[sourceObjectKey],
+		statusKey:       objMap[statusKey],
+		inventoryKey:    objMap[inventoryKey],
```

Comment:
This seems almost exactly redundant with `func (c *combinedStatusResolver) getCombinedContentMap`

Reply:
This was handled.